### PR TITLE
Feat auth

### PR DESCRIPTION
**Pull request by PabloAbara**

### CODE REVIEW

File: src/hooks/useUsersOut.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  const { data, error } = useSWR(`http://localhost:3001/api/sign_out`, fetcher);
+
+  return {
+    data: data,
+    isLoading: !error && !data,
+    isError: error,
+  };
+}
+
+// const fetcher = async (url) => {
```
Comment (AlehandroL) [inline src/hooks/useUsersOut.js]: delete commented out lines

File: src/components/learning-units-list/LearningUnitListItem.js [hunk 2]
```diff
@@ -1,21 +1,42 @@
...
 
+  const handleOnChange = (clicked) => {
+    if (clicked.value) {
+      const requestOptions = {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+      };
+      fetch(`http://localhost:3001/api/learning_units/${unit.id}/completed`, requestOptions)
+        .then((response) => response.json())
+        .then(() => setCompleted(clicked.value));
```
Comment (AlehandroL) [inline src/components/learning-units-list/LearningUnitListItem.js]: then() must use the response as input to change the state

File: src/components/learning-units-list/LearningUnitListItem.js [hunk 3]
```diff
@@ -1,21 +1,42 @@
...
+    } else if (!clicked.value) {
+      const requestOptions = {
+        method: 'DELETE',
+        headers: { 'Content-Type': 'application/json' },
+      };
+      fetch(`http://localhost:3001/api/learning_units/${unit.id}/completed`, requestOptions)
+        .then((response) => response.json())
+        .then(() => setCompleted(clicked.value));
```
Comment (AlehandroL) [inline src/components/learning-units-list/LearningUnitListItem.js]: same

**Review by AlehandroL — CHANGES_REQUESTED** [review]

File: src/hooks/useUsersOut.js (hunk 1 above)
Comment (PabloAbara) [reply src/hooks/useUsersOut.js]: Archivo eliminado

File: src/components/learning-units-list/LearningUnitListItem.js (hunk 2 above)
Comment (PabloAbara) [reply src/components/learning-units-list/LearningUnitListItem.js]: Done

**Review by AlehandroL — APPROVED** [review]

**Pull request merged**